### ISSUE TITLE
Real time inference on Nvidia Jetson?

### ISSUE THREAD

**Issue by YashRunwal** [zylo117/Yet-Another-EfficientDet-Pytorch#615]
Hello,

Firstly, I would like to thank you for making this repo.

I am working on an Object Detection project (multiclass). And after training the EfficientDet architecture on my custom images, I want to deploy the model on one of the **Nvidia Jetsons** (_Not Jetson NANO_). I wanted to know whether it is possible to achieve **real-time inference using EfficientDet on Jetson**. Has anyone done this? 

I have read the paper multiple times and from the graphs, in the paper, I understood that EfficientDet achieves FLOPs of 350-400 billion, fewer than any other architectures. But is this done on a powerful GPU? As I want to infer on Jetson (Specifications: AI Processor - 1.33 TeraFLOPS), will it work the same (i.e. real-time)?

If not, I would like to discuss this further with the changes needed in the model architecture and so on.

**Comment by zylo117** [comment]
I dont think 1.3TFlops device can run effdet in realtime, not even when it's optimized by tensorrt.

**Comment by YashRunwal** [comment]
@zylo117 Thanks for responding. How many TFlops do you think an Embedded device should have in order to run Effdet in Real-time? Or, Can we tweak the architecture in order to make it perform in real-time on the 1.33 TFlops device? Thanks!

**Comment by zylo117** [comment]
it depends.
On dynamic framework like pytorch and its c++ frontend, it takes a 2080ti, which is about 13-14TFlops, to achieve 30fps.

**Comment by YashRunwal** [comment]
Thanks for the reply. Ahh damn. I want to infer real-time on Jetson. Can you perhaps redirect me to a paper or a repo that comes close to this? Or according to you, how should I proceed?

**Comment by zylo117** [comment]
give up accuracy and switch to shallower networks like nanodet
